### PR TITLE
Added polish language files

### PR DESCRIPTION
**Pull request by sdorpl**
initial release polish translate

### CODE REVIEW

**Comment by hakandundar34coding** [comment]
Hello,

Will you translate the remaining texts?

I will merge the pull request after they are translated.

Tomorrow a new release (v1.5.0) will be published. Also translation of another language will be available. Polish translation may be included in v1.5.0 or v1.6.0.

I will write your name to about dialog of the application if you write your name and surname.
Or ```sdorpl``` may be used as translator name.

File: translations/pl/system-monitoring-center.po [hunk 1]
```diff
@@ -0,0 +1,1711 @@
...
+#: src/UsersDetails.py:226
+msgid "No"
+msgstr "Nie"
+
+#: src/Disk.py:299 src/Disk.py:300 src/Disk.py:301 src/Disk.py:302
+#: src/DiskDetails.py:165 src/DiskDetails.py:195 src/DiskDetails.py:200
+#: src/DiskDetails.py:224 src/DiskDetails.py:351 src/DiskDetails.py:352
+#: src/DiskDetails.py:353 src/DiskDetails.py:354
+msgid "[Not mounted]"
+msgstr "[Nie zamontowany]"
```
Comment (K0-RR) [inline translations/pl/system-monitoring-center.po:51]: This should be written together

File: translations/pl/system-monitoring-center.po [hunk 2]
```diff
@@ -0,0 +1,1711 @@
...
+msgid "Graphical User Interface (GUI)"
+msgstr "Interfejs graficzny (GUI)"
+
+#: ui/SystemTab.ui:568
+msgid "Packages"
+msgstr "Pakiety"
+
+#: ui/SystemTab.ui:585
+msgid "Desktop Environment:"
+msgstr "Pulpit:"
```
Comment (K0-RR) [inline translations/pl/system-monitoring-center.po:1649]: Too vague, it's not Windows. IMO it should be "Środowisko Pulpitu"

File: translations/pl/system-monitoring-center.po [hunk 3]
```diff
@@ -0,0 +1,1711 @@
...
+msgid "IPv6 Address:"
+msgstr "Adres IPv6:"
+
+#: ui/NetworkTab.ui:120
+msgid "MAC Address:"
+msgstr "Adres MAC:"
+
+#: ui/NetworkTab.ui:269
+msgid "Connected-SSID:"
+msgstr "Połączono z:"
```
Comment (K0-RR) [inline translations/pl/system-monitoring-center.po:1098]: More technical translation would be better - "SSID połączenia:"?

**Comment by K0-RR** [comment]
I have briefly looked at this translation and it's good. If @sdorpl won't continue the translation I can do it. Waiting for response.

**Comment by hakandundar34coding** [comment]
It has been about 4 days since the last commit. I do not know if @sdorpl will translate rest of the text.

I will merge this pull request.

@RDKRACZ will you translate rest of the text?

I will add both your name (if you write) and @sdorpl 's name (if writes to me) into about dialog of the application. Because @sdorpl has translated some of the text.

You can also translate 4 lines in the ```.desktop``` file. Content of this file is used for name/explanation/keywords of the application on the system start menu, etc.

Here is the content of the ```.desktop``` file:

```
[Desktop Entry]
Name=System Monitoring Center
Name[tr]=Sistem Gözlem Merkezi
GenericName=System Monitor
GenericName[tr]=Sistem Gözlemcisi
Comment=Gets information about system performance and usage.
Comment[tr]=Sistem performansı ve kullanımı hakkında bilgi verir.
Exec=system-monitoring-center
Icon=system-monitoring-center
Terminal=false
Type=Application
Categories=System;
StartupNotify=true
Keywords=system monitor;task manager;center;performance;speed;frequency;cpu usage;ram usage;swap memory;memory usage;storage;network usage;download speed;fps;frame ratio;processes;users;startup programs;services;os;
Keywords[tr]=sistem gözlemcisi;görev yöneticisi;performans;hız;frekans;cpu;mib;ram kullanımı;takas belleği;bellek kullanımı;depolama;indirme hızı;fps;kare oranı;işlemler;kullanıcılar;başlangıç programları;servisler;os;is;
```

Note: Some texts such as ```Connected-SSID:``` has ```-``` character and this means there are two different information. For this example these are:
- The information of if the card is connected (Yes or No)
- The information of SSID (network name).

A similar character may be used for translation of the texts.

**Pull request merged**